microsoft-edge/webdriver-chromium/ie-mode.md

```diff
@@ -0,0 +1,376 @@
+
+```csharp
+//change the path accordingly
+var dir = @"C:\<path to IEDriver download>";
+var driverexe = "IEDriverServer.exe";
+
+var ieService = InternetExplorerDriverService.CreateDefaultService(dir, driverexe);
+```
+
+### [Python](#tab/python/)
+
+Define `ieService` by calling `Service` from `selenium.webdriver.ie.service` with the path to the IEDriver executable you downloaded.
+
+```python
+from selenium.webdriver.ie.service import Service
+
+ieService = Service('C:/<path to IEDriver download>')
+```
+
+### [Java](#tab/java/)
+
+For Java, you do not need to create an InternetExplorerDriverService. You do need to specify the path to the IEDriver executable you downloaded.
+
+```java
+System.setProperty("webdriver.ie.driver","C:/<path to IEDriver download>");
+```
+
+### [JavaScript](#tab/javascript/)
+
+Define `ieService` by calling `ServiceBuilder` and passing it the path to the IEDriver executable you downloaded.
+
+```javascript
+let ieService = await new ServiceBuilder('C:/<path to IEDriver download>')
+```
+
+* * *
+
+
+<!-- ====================================================================== -->
+## Define InternetExplorerOptions with additional capabilities for Microsoft Edge
+
+### [C#](#tab/c-sharp/)
+
+1. Define a new variable, `ieOptions`, by instantiating `InternetExplorerOptions` with the `IgnoreZoomLevel` property set to `true`.
+
+1. Set `ieOptions.AttachToEdgeChrome` property to `true`, and `ieOptions.EdgeExecutablePath` to the path of the Microsoft Edge executable:
+
+```csharp
+var ieOptions = new InternetExplorerOptions { IgnoreZoomLevel = true };
+ieOptions.AttachToEdgeChrome = true;
+//change the path accordingly
+ieOptions.EdgeExecutablePath = "C:/Program Files (x86)/Microsoft/Edge/Application/msedge.exe";
+```
+
+### [Python](#tab/python/)
+
+1. Define a new variable, `ieOptions`, by calling `webdriver.IeOptions()`.
+
+1. Set the `ieOptions.attach_to_edge_chrome` property to `true`, and `ieOptions.edge_executable_path` to the path of the Microsoft Edge executable:
+
+```python
+ieOptions = webdriver.IeOptions()
+ieOptions.attach_to_edge_chrome = True
+ieOptions.edge_executable_path = "C:/Program Files (x86)/Microsoft/Edge/Application/msedge.exe"
+```
+
+### [Java](#tab/java/)
+
+1. Define a new variable `ieOptions` of type `InternetExplorerOptions`, by calling `new InternetExplorerOptions()`.
+
+1. Call `ieOptions.attachToEdgeChrome()` and `ieOptions.withEdgeExecutablePath()` with the path of the Microsoft Edge executable:
+
+```java
+InternetExplorerOptions ieOptions = new InternetExplorerOptions();
+ieOptions.attachToEdgeChrome();
+ieOptions.withEdgeExecutablePath("C:\\Program Files (x86)\\Microsoft\\Edge\\Application\\msedge.exe");
+```
+
+### [JavaScript](#tab/javascript/)
+
+1. Define a new variable, `ieOptions`, by calling `Options()`.
+
+1. Set the `ieOptions.setEdgeChromium` property to `true` and `ieOptions.setEdgePath` to the path of the Microsoft Edge executable:
+
+```javascript
```

Comment:
Global suggestion: Consistently use or omit ; for JS code lines. I'd probably include ; in JS lines, for clarity.